.github/workflows/code-review.yml

```diff
@@ -16,7 +11,6 @@ permissions: write-all
 jobs:
   review:
     runs-on: ubuntu-latest
-    if: ${{ contains(github.event.pull_request.labels.*.name, 'requires-review') }}
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
```

Comment:
You have removed the condition `if: ${{ contains(github.event.pull_request.labels.*.name, 'requires-review') }}`. If you want the job to run only when the pull request has a specific label, you should add this condition back.